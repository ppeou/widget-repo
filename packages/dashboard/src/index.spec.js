import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Component from '.';

describe('Dashboard', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Component>{'Test'}</Component>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});