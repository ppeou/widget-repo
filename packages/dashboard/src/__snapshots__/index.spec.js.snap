// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders correctly 1`] = `
<section>
  <h1>
    Dashboard Component
  </h1>
</section>
`;
